src/app/services/http.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class HttpService {
+  constructor(private http: HttpClient) {}
+
+  public list<T>(url: string): Observable<T> {
+    return this.http.get<T>(url);
+  }
+
+  public get<T>(url: string): Observable<T> {
```

Comment:
maybe, it make sense pass id to get, because we already have method with same implementation, or left only 1 get or list